VisualStudio/Docs/run.md

```diff
@@ -17,6 +17,7 @@ and can be omitted).
 Examples:
 
 ```
+<!-- Examples - do not remove this line because it's used by the console app to render the examples -->
```

Comment:
I'd make this more obviously pragma-like, i.e. `<!-- EXAMPLES_BEGIN -->` and the matching `END`

Reply:
I would avoid the Begin/End to simplify the logic I already have. Initially I was planning to "parse" the ``` content ``` but you might also use that for other sections in the md. But I would follow the same concept of using the same text for the begin/end  "token"